tests/integration/tests.py

```diff
@@ -168,21 +169,29 @@ def queue_event(method, url, body, headers):
         assert group.data["title"] == "foo"
 
 
-class SentryRemoteTest(TestCase):
+class SentryRemoteTest(SnubaTestCase):
     @fixture
     def path(self):
         return reverse("sentry-api-store")
 
+    def get_event(self, event_id):
+        instance = eventstore.get_event_by_id(self.project.id, event_id, eventstore.full_columns)
+        Event.objects.bind_nodes([instance], "data")
```

Comment:
When are we supposed to use the bind_node method on eventstore and when should we use the one on the Event class ?

Reply:
Right now, not yet using the eventstore one anywhere